miLazyCracker.sh

```diff
@@ -5,7 +5,7 @@
 # gets a valid answer.
 
 ask() {
-    # http://djm.me/ask
+    # https://djm.me/ask
```

Comment:
you can remove entirely this one, the URL doesn't exist anymore

Reply:
Done. 😊